src/mixins/numeral-formatable.js

```diff
@@ -90,66 +90,84 @@ export default {
 
       this.sign = text = String(text)
       text = this.internalChange ? this.removeNonNumeral(text) : this.attemptNumeralCorrection(text)
-      if (!text) return text // attemptNumeralCorrection() may return null or falsy
 
       return this.parseNumber(text)
     },
     maskNumeralText (text) {
       if (!text) return this.numeralZero()
       text = this.sign ? String(text).substr(1) : String(text)
 
-      const masked = []
-      let i = this.blockSeparator.length - 1
-      let j = this.blockSize.length - 1
-      let length = 0
-      let integer = !this.precision
+      const param = {
+        masked: [],
+        i: this.blockSeparator.length - 1,
+        j: this.blockSize.length - 1,
+        length: 0,
+        integer: !this.precision
+      }
 
-      // Group the digits with specified separators.
-      // The first size and separator element
-      // of the arrays becomes the default value.
+      // Group the digits with specified separators and digit counts.
       for (const digit of text.split('').reverse()) {
-        digit !== '.' && masked.push(digit)
-        if (integer && !(++length % this.blockSize[j])) {
-          masked.push(this.oneChar(this.blockSeparator[i], ' '))
-          i > 0 && i--
-          j > 0 && j--
-          length = 0
-        } else if (digit === '.') {
-          masked.push(this.decimal)
-          integer = true
+        if (!this.maskNumeral(digit, param)) break
```

Comment:
`maskNumeral` always returns `true`, so this line (as well as `return true` in `maskNumeral`) can be removed

Reply:
The return `false` is at line 123.

Reply:
How did I miss it...

Reply:
😄